src/state/metadata/actions.ts

```diff
@@ -28,19 +29,23 @@ import {
     ReceiveAlbumDataAction,
     ReceiveAvailableDatasetsAction,
     ReceiveCellFileInfoAction,
+    ReceiveImageDatasetAction,
     ReceiveMeasuredFeaturesAction,
     ReceiveViewerChannelSettingsAction,
     RequestAction,
     SetLoadingAction,
     SetSmallScreenWarningAction,
 } from "./types";
 
+export function replaceImageDataset(payload: ImageDataset): ReceiveImageDatasetAction {
+    return { payload, type: REPLACE_IMAGE_DATASET };
+}
```

Comment:
Nit: `ReceiveImageDatasetAction` and `REPLACE_IMAGE_DATASET` should be the same

Reply:
I think I resolved this? LMK if this looks how you'd expect in the new `image-dataset/actions.ts` file